docker/launch.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ -z "$FORK" ]]; then
+    FORK=PowerShell
+fi
+
+if [[ -z "$BRANCH" ]]; then
+    BRANCH=master
+fi
+
+# Build both sets by default
+if [[ -z "$BUILDS" ]]; then
+    BUILDS="stable unstable"
+fi
+
+# Build specified distributions
+if [[ -z $DISTROS ]]; then
+    DISTROS="ubuntu14.04 ubuntu16.04 centos7"
+fi
+
+for build in $BUILDS; do
+    # cd so $distro is only the distro name
+    cd $build
+    for distro in $DISTROS; do
+        image="$build/$distro"
+        if [[ "$TEST" -eq 1 ]]; then
+            echo "Testing $image"
+            command="cd PowerShell; Import-Module ./build.psm1; Install-Dotnet -NoSudo; Start-PSPester -powershell powershell -Pester ./src/Modules/Shared/Pester"
+            # clone repo for stable images because it's not already done
+            if [[ "$build" = stable ]]; then
+                command="git clone --recursive https://github.com/$FORK/PowerShell -b $BRANCH; $command"
+            fi
+            # run Pester tests inside container
+            docker run --rm -it powershell/powershell:$build-$distro -c "$command"
```

Comment:
These tests *do not all* pass, and it is outside the scope of this PR to make them pass.

/cc @JamesWTruher these are the 1% I mentioned